make/migrations/postgresql/0030_1.11.0_schema.up.sql

```diff
@@ -0,0 +1,40 @@
+/* TODO remove the table artifact_2 and use the artifact instead after finishing the upgrade work */
+CREATE TABLE artifact_2
+(
+  id            SERIAL PRIMARY KEY NOT NULL,
+  /* image, chart, etc */
+  type          varchar(255),
+  media_type    varchar(255),
+  manifest_media_type varchar(255),
+  project_id    int NOT NULL,
+  repository_id int NOT NULL,
+  digest        varchar(255) NOT NULL,
+  size          bigint,  
+  push_time     timestamp default CURRENT_TIMESTAMP,
+  platform      varchar(255),
+  extra_attrs   text,
+  annotations   text,
+  revision      int NOT NULL,
```

Comment:
let's use varchar to store the revision and explicitly document that when updating the data the revision MUST be checked and updated.

Reply:
Updated